blog/electron-16-0.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Electron 16.0.0
+date: 2021-11-16T00:00:00.000Z
+authors:
+    - name: sofianguy
+      url: 'https://github.com/sofianguy'
+      image_url: 'https://github.com/sofianguy.png?size=96'
+    - name: ckerr
+      url: 'https://github.com/ckerr'
+      image_url: 'https://github.com/ckerr.png?size=96'
+slug: electron-16-0
+
+---
+
+Electron 16.0.0 has been released! It includes upgrades to Chromium `96`, V8 `9.6`, and Node.js `16.9.1`. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 16.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://www.electronjs.org/releases/stable). Continue reading for details about this release and please share any feedback you have!
+
+## Notable Changes
+
+### Electron Release Cadence Change
+
+Starting with Electron 15, Electron will release a new major stable version every 8 weeks. You can read the [full details here](https://www.electronjs.org/blog/8-week-cadence).
```

Comment:
Can we modify the verb tense going forward because we're no longer talking about the future?